### ISSUE TITLE
Color contrast in tables

### ISSUE THREAD

**Issue by chiaravicere** [grabs/moodle-local_adminer#29]
Hello, in our version the database tables have become difficult to read, due to a poor color contrast between the black table background and the dark grey text for the values. It does not seem to be theme specific, it is the same with Boost theme activated.

Plugin version: 5.2.0 (2025110700)
Moodle version: 4.5.4+ (Build: 20250509)

<img width="751" height="276" alt="Image" src="https://github.com/user-attachments/assets/a433bc16-22ec-444f-adcb-48284a7c95c4" />

**Comment by stdoma** [comment]
I noticed the same thing / problem.
I installed the latest version of Adminer today.

**Comment by grabs** [comment]
I fixed the adminer to the light theme in the last release. So now all should be visible in the right contrast.
Thanks for reporting.
